test/onnx/onnx_import.in.cpp

```diff
@@ -355,6 +355,21 @@ NGRAPH_TEST(onnx_${BACKEND_NAME}, model_initializer_wo_input)
     EXPECT_TRUE(test::all_close_f(expected_output, output.front()));
 }
 
+NGRAPH_TEST(onnx_${BACKEND_NAME}, provenance_tag_text)
+{
+    auto function = onnx_import::import_onnx_model(
+        file_util::path_join(SERIALIZED_ZOO, "onnx/provenance_tag_add.prototxt"));
+
+    auto ng_nodes = function->get_ordered_ops();
+    for (auto ng_node : ng_nodes)
+    {
+        for (auto tag : ng_node->get_provenance_tags())
+        {
+            EXPECT_EQ("<ONNX Add (add_node1)>", tag);
```

Comment:
Maybe we could be here less restrictive? If provenance tag formatting would change, then this condition fail. I'd suggest using macro `EXPECT_HAS_SUBSTRING` from `test/util/type_prop.hpp` file.  And set condition to *ONNX Add*.
https://github.com/NervanaSystems/ngraph/blob/731959f749ce059b5bb2bec8e028d45a7223a9fb/test/util/type_prop.hpp#L21

Reply:
Good idea, done.